src/m4a_tables.c

```diff
@@ -243,7 +245,7 @@ const u8 gClockTable[] =
 #define BENDR  0xc1
 #define LFOS   0xc2
 #define LFODL  0xc3
-#define MOD    0xc4
+#define MOD  0xc4
```

Comment:
I don't think you meant to commit this change?

Reply:
will revert